docker-compose.yml

```diff
@@ -1,16 +1,16 @@
-version: '3.7'
+version: "3.7"
 services:
-    postgres:
+    palvelutarjotin-db:
```

Comment:
I'd prefer to use `kultus-db` here. `palvelutarjotin` naming is deprecated so better to avoid using it. Feel free to replace other `palvelutarjotin` with `kultus` among the rest of this file.

Reply:
It will look a bit messy:
![image](https://user-images.githubusercontent.com/389204/104215361-4ad4ce80-5441-11eb-973c-9628ca625943.png)

However, it's a step to a right direction. It's now done and tested:
1. Removed all old containers
2. docker-compose up --build

Result: all running well